app/controllers/solidus_klarna_payments/callbacks_controller.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
 module SolidusKlarnaPayments
-  class CallbacksController < ::Spree::StoreController
+  class CallbacksController < ::Spree::BaseController
+    include ::Spree::Core::ControllerHelpers::Order
```

Comment:
Why do we need to include this? Do we need to set the `current_order` for these actions?

Reply:
we don't need it. I have removed it.